pkg/test/store/load.go

```diff
@@ -73,7 +74,7 @@ func ExtractResources(ctx context.Context, rs store.ResourceStore) (string, erro
 		items := slices.SortedFunc(
 			slices.Values(resList.GetItems()),
 			func(a, b model.Resource) int {
-				return cmp.Compare(a.GetMeta().GetName(), b.GetMeta().GetName())
+				return strings.Compare(a.GetMeta().GetName(), b.GetMeta().GetName())
```

Comment:
is there a difference?

Reply:
I wasn't sure and since i did a lot of runs locally with this setup I left it

Reply:
Above uses cmp